app/src/main/res/layout/fragment_share.xml

```diff
@@ -54,6 +54,14 @@
             android:background="?borderPrimary"
             app:layout_constraintBottom_toTopOf="@id/appsShareLayout" />
 
+        <androidx.compose.ui.platform.ComposeView
+            android:visibility="gone"
+            android:id="@+id/save_pdf"
+            android:layout_width="match_parent"
+            android:layout_height="72dp"
```

Comment:
This 72dp conflicts with the sizes in the composable dimensions (48dp in the code, but this is also wrong from the designs). It looks a bit off on my device with asymmetric padding (more on the bottom).

```suggestion
            android:layout_height="wrap_content"
```